src/main/resources/db/migration/V6__create_job_role_table.sql

```diff
@@ -1,18 +1,39 @@
 CREATE TABLE IF NOT EXISTS JobRoles (
     jobRoleId TINYINT NOT NULL AUTO_INCREMENT,
     roleName VARCHAR(64) NOT NULL,
+    `description` VARCHAR(500) NOT NULL,
+    responsibilities VARCHAR(500) NOT NULL,
     location VARCHAR(50) NOT NULL,
     bandId TINYINT NOT NULL,
     capabilityId TINYINT NOT NULL,
     closingDate DATETIME NOT NULL,
-    status BOOLEAN NOT NULL,
+    `status` BOOLEAN NOT NULL,
+    specification VARCHAR(300) NOT NULL,
+    CONSTRAINT specification_check CHECK (specification REGEXP 'https://'),
```

Comment:
don't edit existing migrations as they will already have run on production. instead create a new migration to alter the table

Reply:
I added extra migration files like you asked, the information in the insert statements didn't match any share point document so I had to redo them- they are in a new migration file 
1. I added an extra migration file for the alter statements to add description, responsibilities and specification 
2. I added an extra migration file in for the insert statements 
is this okay?